### DISCUSSION TITLE
send to controller - single stepping?

### DISCUSSION THREAD

**Discussion by phil-barrett** [terjeio/ioSender#351] [General]
The new 2.0.44 release looks good!  Thanks for releasing it.

The new feature:

>  Added right click menu in the program listing to start from a given block, execute a single block or send selected commands to the controller.

I like that you can even send a set of selected blocks to the controller. Though, it does beg the question about single stepping. It might be useful for debugging mechanical or setup problems.

**Comment by terjeio** [comment]
Try `$S` - currently undocumented but should toggle single stepping. I need to complete aux input support for this and similar signals...
